jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/filter/SingleFilterComponentBase.java

```diff
@@ -203,12 +204,16 @@ public Condition getQueryCondition() {
     public void apply() {
         if (dataLoader != null) {
             setupLoaderFirstResult();
-            if (autoApply) {
+            if (shouldAutoApply() && autoApply) {
```

Comment:
is `SingleFilterComponent` is the problem? For example, if I have several GenericFilter configurations saved, when I open a view, the number of data refreshes is comparable to the number of configurations, not the number of conditions.